lib/percy/capybara/rspec.rb

```diff
@@ -1,6 +1 @@
 require 'rspec/core'
-
-RSpec.configure do |config|
```

Comment:
This needs to remain for backwards compatibility — it's not documented anywhere as we removed it and have encouraged people to move away from this old method, but people might still have a `require 'percy/capybara/rspec` line somewhere and we can't get rid of this until the next major version bump (which we could do, but later).